### ISSUE TITLE
NullReferenceException error when enabling P mode in non-finished levels

### ISSUE THREAD

**Issue by Katzenwerfer** [Hydraxous/EasyPZ-ULTRAKILL#1]
```
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
EasyPZ.EasyPZUIPatch.UpdateTracker () (at <9a8452fea00c4273a0bbc86c06a04235>:0)
EasyPZ.EasyPZUIPatch.Update () (at <9a8452fea00c4273a0bbc86c06a04235>:0)
```

**Comment by Hydraxous** [comment]
Thank you for the bug report.
EasyPZ will be rewritten from the ground up soon, this should be fixed when that happens.

**Comment by Hydraxous** [comment]
This should be resolved as of 3.0.0
